src/main/steps/sequence.ts

```diff
@@ -126,13 +126,10 @@ export const sequence: Step[] = [
   {
     url: HELP_WITH_YOUR_FEE_URL,
     showInSection: Sections.HelpWithFees,
-    getNextStep: (data: Partial<CaseWithId>): PageLink => {
-      if (data.applicationType === ApplicationType.JOINT_APPLICATION) {
-        return CHECK_ANSWERS_URL;
-      } else {
-        return data.applicant1HelpPayingNeeded === YesOrNo.YES ? HELP_PAYING_HAVE_YOU_APPLIED : IN_THE_UK;
-      }
-    },
+    getNextStep: data =>
+      data.applicationType === ApplicationType.JOINT_APPLICATION || data.applicant1HelpPayingNeeded === YesOrNo.YES
+        ? HELP_PAYING_HAVE_YOU_APPLIED
+        : IN_THE_UK,
```

Comment:
I think it could just be 
```suggestion
getNextStep: data => (data.applicant1HelpPayingNeeded === YesOrNo.YES ? HELP_PAYING_HAVE_YOU_APPLIED : IN_THE_UK),
```

Reply:
That's what's implied by my ticket - https://tools.hmcts.net/jira/browse/NFDIV-654

Reply:
true, looking at both journeys on the miro board, both go to the same place so we dont need to check. i just continued from what was there.